### PR TITLE
.NET application site fix (Part - 6)

### PR DESCRIPTION
**Pull request by omar-srbd**
### Change Description ###

This is the sixth PR in the process of updating the .NET applications section of Tizen Docs site.

This PR includes the fix of the following topics:

1. UI Application **[File count: 1]**
2. Connectivity and Wireless **[File count: 10]**

List of files on which changes were made:

- /application/dotnet/guides/applications/uiapplication/component-based-app/
- /application/dotnet/guides/connectivity/overview/
- /application/dotnet/guides/connectivity/connection/
- /application/dotnet/guides/connectivity/nsd/
- /application/dotnet/guides/connectivity/wifi/
- /application/dotnet/guides/connectivity/wifi-direct/
- /application/dotnet/guides/connectivity/bluetooth/
- /application/dotnet/guides/connectivity/nfc/
- /application/dotnet/guides/connectivity/smartcard/
- /application/dotnet/guides/connectivity/stc/
- /application/dotnet/guides/connectivity/iotcon/

Type of fixes made on the documents:

- Gerund form on headings
- Sentence case on headings
- Incomplete sentence preceding a list
- Missing comma before code snippets
- API hyperlinks fix

Signed off by: omar.saeed@samsung.com

### CODE REVIEW

File: docs/application/dotnet/guides/applications/uiapplication/component-based-app.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ Following are the main application attributes:
 
 ## Prerequisites
 
-To use the functions and data types of the component based application API (in [mobile](/application/dotnet/api/mobile/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) and [wearable](/application/dotnet/api/wearable/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) applications), include the `Tizen.Applications.ComponentBased.Common` module in your application:
+To use the functions and data types of the [Tizen.Applications.ComponentBased.Common](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.Common.html) namespace, include it in your application:
```
Comment (omar-srbd) [inline docs/application/dotnet/guides/applications/uiapplication/component-based-app.md:311]: Hello @mijin85cho, hope you're doing well.

As per our discussion on https://github.com/Samsung/tizen-docs/pull/1734#discussion_r1045541882, I have made the necessary change.

Please check the change made, thank you.

**Review by safir-srbd — COMMENTED** [review]
@omar-srbd I have provided some suggestions. Please implement them at your earliest convenience.

File: docs/application/dotnet/guides/connectivity/wifi.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
     - [Activate](#activating_wifi_device) or [deactivate](#deactivating_wifi_device) Wi-Fi, [monitor connection state changes](#managing_events), and [scan for available access points](#accesspoint_scan).
     - Get BSSID asynchronously as well as synchronously.
-    - Make connections between an access point(AP) and wireless devices faster and easier using Wi-Fi Protected Setup (WPS).
+    - Make connections between an access point (AP) and wireless devices faster and easier using Wi-Fi Protected Setup (WPS).
 
 - Access point management
 
-    With the [Tizen.Network.WiFi.WiFiAP](/application/dotnet/api/TizenFX/latest/api/Tizen.Network.WiFi.WiFiAP.html) class, you can [connect to a specific access point](#accesspoint_connect) and manage the Wi-Fi access point and security information:
+    With the [Tizen.Network.WiFi.WiFiAP](/application/dotnet/api/TizenFX/latest/api/Tizen.Network.WiFi.WiFiAP.html) class, you can [connect to a specific access point](#accesspoint_connect) and manage the Wi-Fi access point and security information, follow these steps to manage Wi-Fi access point and security information:
```
[the remark this answers is not in the document]
Comment (omar-srbd) [reply docs/application/dotnet/guides/connectivity/wifi.md]: @mijin85cho hello,
I have made the appropriate change to this sentence in my most recent commit.
Please have a check.
Thank you.

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/guides/connectivity/wifi.md]: @omar-srbd I checked that my comment was resolved. 
Thanks.

**Review by safir-srbd — APPROVED** [review]
@omar-srbd Thank you for implementing all the suggested changes. Everything seems to be in order.

@asm-foysal @mijin85cho I am approving this PR for merging.

File: docs/application/dotnet/guides/connectivity/smartcard.md [hunk 3]
```diff
@@ -84,9 +84,9 @@ public static void GetReaders_RETURN_LIST_OF_READERS()
 ```
 
 <a name="reader"></a>
-## Managing the Reader
+## Manage the Reader
```
Comment (asm-foysal) [inline docs/application/dotnet/guides/connectivity/smartcard.md:87]:
```suggestion
## Manage the reader
```

**Pull request merged**